source/_integrations/imap.markdown

```diff
@@ -40,15 +40,19 @@ By default, this integration will count unread emails. By configuring the search
 * `FROM`, `TO`, `SUBJECT` to find emails in a folder (see [IMAP RFC for all standard options](https://tools.ietf.org/html/rfc3501#section-6.4.4))
 * [Gmail's IMAP extensions](https://developers.google.com/gmail/imap/imap-extensions) allow raw Gmail searches, like `X-GM-RAW "in: inbox older_than:7d"` to show emails older than one week in your inbox. Note that raw Gmail searches will ignore your folder configuration and search all emails in your account!
 
+If you run into an error saying the selected search is invalid, that could be the case or you need to use a different character set.
```

Comment:
The error and possible solution seems too vague in my opinion. I would leave this bit out and keep your note below on setting the character set for Yahoo.

Reply:
removed the note